doc_admin/AdminManual.adoc

```diff
@@ -49,50 +50,117 @@ Open Hospital features include:
 
-For the MariaDB/MySQL database server installation and configuration refer to the chapter <<database-server,Database Server>> and the online documentation.
+Open Hospital does not require installation. It is sufficient to unzip the downloaded package in a folder of choice, e.g.
 
-For the OH database creation refer to the chapter <<ohdatabase-creation,Open Hospital - Database creation>> and the online documentation.
+* on Linux:
+----
+/home/OH/OpenHospital-[VERSION]
+----
+
+* on Windows:
+----
+C:\Users\OH\OpenHospital-[VERSION]
+----
 
-For the Open Hospital installation on the clients, it is sufficient to copy and extract the zip package in the desired folder.
+To uncompress the package:
 
-For the Open Hospital configuration refer to the chapter <<config-files,Configuration Files>>.
+* on Linux:
+----
+cd /home/OH
+tar zxvf OpenHospital-v[VERSION]-linux_[ARCH]-[portable].tgz # portable version
+unzip x OpenHospital-v[VERSION]-linux_multiarch.zip # multiarch client version
+----
+* on Windows:
+----
+unzip OpenHospital-v[VERSION]-windows_[ARCH]-[portable].zip # portable version
+unzip OpenHospital-v[VERSION]-multiarch-client.zip à multiarch client version
+----
 
+After uncompressing the package, browse to the extracted directory (example given for version 1.11.1):
 
-==== 3.1.1 Prerequisites - Software compatibility matrix
+* on Linux:
+----
+cd /home/OH/OpenHospital-v1.11.1
+----
+* on Windows:
+----
+cd C:\Users\OH\OpenHospital-v1.11.1
+----
+
+
+NOTE: You need read/write permission on the selected folder.
+
+NOTE: Do not run OH by double-clicking on the oh.bat startup script without unzipping the package first!
+
+=== 1.7 Documentation
+
+In the following chapters, all the information needed to install, configure, deploy, run and maintain an Open Hospital 
+installation is presented, including procedures on how to enable and disable features as well as managing users and groups in a multi user environment.
+More information can be found on the Open Hospital web site https://www.open-hospital.org.
+
+NOTE: The information needed to use the OH software is not included in this manual; please refer to the _Open Hospital User’s Guide_.
+
+
+== 2 Installation
+
+=== 2.1 OH - CLIENT mode
+
+Open Hospital is a client application that can be used on a single computer (PORTABLE mode) or in a client/server configuration (CLIENT mode). The server part utilizes a DBMSfootnote:[DataBase Management System] to archive, share, and present the information. The free and open-source MariaDBfootnote:[See https://www.mariadb.org] is the database server of choice.
+
+The typical client/server, networked (LAN) configuration is shown in the following figure:
+
+image:image3.png[image,width=642,height=243]
+
+The Administrator/Installer tasks are hereby listed:
+
+* <<download,Download>> the Open Hospital package for the Operating System of choice
+* <<setup,Setup>> the downloaded OH package by uncompressing the archive in a folder of choice
```

Comment:
again these two lines I would add "your" before "choice";  probably just my crazy preferences

Reply:
Should I replace all occurences of these sentence ?